examples/model/app.py

```
@@ -0,0 +1,40 @@
+__copyright__ = "Copyright (c) 2008-2024 Hive Solutions Lda."
+""" The copyright for the module """
+
+__license__ = "Apache License, Version 2.0"
+""" The license for the module """
+
+import appier
+
+
+class ModelApp(appier.App):
+    def __init__(self, *args, **kwargs):
+        appier.App.__init__(self, name="model", *args, **kwargs)
```

Ruff (0.12.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)